docs/rules/arrow-parens.md

```diff
@@ -87,7 +87,7 @@ a.then((foo) => { if (true) {}; });
 
 #### If Statements
 
-One benefits of this option is that it prevents the incorrect use of arrow functions in conditionals:
+One of benefits of this option is that it prevents the incorrect use of arrow functions in conditionals:
```

Comment:
Good point - is it OK if that change is done at a later date (I'll make note of it as separate ticket)?

Reason: Currently my focus is on ensuring there are consistent headings for examples, and also ensuring that the captions for examples are picked up by https://github.com/eslint/eslint/issues/6444#issuecomment-226856201 which in turn will colorize and iconify the examples.

I plan to do the edits to the remainder of the docs in subsequent sweeps: https://github.com/eslint/eslint/issues/6444#issue-160784214